cdap-app-fabric/src/test/java/co/cask/cdap/metadata/MetadataHttpHandlerTest.java

```diff
@@ -809,6 +809,76 @@ public void testScopeQueryParam() throws Exception {
   }
 
   @Test
+  public void testSearchTargetType() throws Exception {
+    Id.Namespace namespace = Id.Namespace.from("testSearchTargetType");
```

Comment:
can we use ``NamespaceId`` for all new uses? Can easily convert it to ``Id.Namespace`` by doing a ``toId()``